### PR TITLE
RedisStore use ClientInterface instead of Client

### PR DESCRIPTION
**Pull request by fhjbalfoort**
We use RedisStore in combination with Predis. We wrap the default client with metric but RedisStore doesn't accept a ClientInterface so we are stuck with passing a Client.

I changed the RedisStore so that it will accept a ClientInterface instead of a Client

### CODE REVIEW

File: src/Ganesha/Storage/Adapter/RedisStore.php [hunk 1]
```diff
@@ -178,7 +178,7 @@ public function get(string $key)
     {
         try {
             $result = $this->redis->get($key);
-            if ($this->redis instanceof \Predis\Client && $result === null) {
+            if ($this->redis instanceof \Predis\ClientInterface  && $result === null) {
```
Comment (ackintosh) [inline src/Ganesha/Storage/Adapter/RedisStore.php:181]:
```suggestion
            if ($this->redis instanceof \Predis\ClientInterface && $result === null) {
```

**Review by ackintosh — COMMENTED** [review]
@fhjbalfoort Thank you for your PR. This looks great to me!

Although the code style check in CI has passed, I've left a tiny suggestion.

File: src/Ganesha/Storage/Adapter/RedisStore.php (hunk 1 above)
Comment (fhjbalfoort) [reply src/Ganesha/Storage/Adapter/RedisStore.php]: Thank you for reviewing and updated the MR with your suggestion

**Review by ackintosh — APPROVED** [review]
Looks good to me. Thank you!

**Comment by ackintosh** [comment]
This pull request has been released as v2.0.2. 🎉 
https://github.com/ackintosh/ganesha/releases/tag/2.0.2

**Pull request merged**